### DISCUSSION TITLE
Stop Button - Fixed

### DISCUSSION THREAD

**Discussion by nnn19** [VarunAgw/CloudPingTest#5] [Ideas] [answered]
Better to keep the Stop button in an Absolute or Fixed position - may be on right top of the screen always.

(As I could not add Comment to the existing Idea Discussion, I started a new Idea).

**Comment by VarunAgw** [comment]
Duplicate of https://github.com/VarunAgw/CloudPingTest/discussions/2

**Comment by VarunAgw** [comment]
I am not sure why you unable to add a comment to it

**Comment by nnn19** [comment, marked as answer]
Yesterday, while Comment button is clicked, an error message
You can not comment...this time - an alert box in red.

But now, it accepts - and the same comment is added there.
